### PR TITLE
docs(README): update TS example

### PR DESCRIPTION
**Pull request by jsjoeio**
## Description

Updates the TypeScript example
## Motivation and Context

Makes it up to date.

## How has this been tested?

Tested locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

<a href="https://gitpod.io/#https://github.com/chimurai/http-proxy-middleware/pull/800"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ app.listen(3000);
 ```typescript
 // typescript
 
-import * as express from 'express';
-import { createProxyMiddleware, Filter, Options, RequestHandler } from 'http-proxy-middleware';
```
Comment (chimurai) [inline README.md:55]: Why are the `types` removed?

Comment (jsjoeio) [reply README.md:55]: Because they're not used in the example

**Comment by jsjoeio** [comment]
> The proposed change is dependent on the settings of your tsconfig.json.
These is no right or wrong. Really depends on your project...

Ah well I was just trying to make it easier to copy & paste the TS example from the `README`. If you disagree, feel free to reject PR and close

**Comment by chimurai** [comment]
keeping types as part of documentation

**Pull request closed without merging**